pkg/packaging/packaging_test.go

```diff
@@ -136,6 +139,67 @@ func TestHelperProcess(t *testing.T) {
 
 }
 
+func Test_getBinary(t *testing.T) {
+	t.Parallel()
+
+	if runtime.GOOS == "windows" {
+		t.SkipNow()
+	}
```

Comment:
Why skip this on windows?

Reply:
I wonder if it's something simple, like the fetcher not realizing it's a local path and trying to download from notary

Reply:
Oh, I remember now. So [the check for the local path](https://github.com/kolide/launcher/blob/main/pkg/packaging/packaging.go#L342) checks for `binaryVersion` (i.e. our cached location) to start with `./` or `/`. On Windows, though, it starts with `C:\`, when then would force the test to go through the `FetchBinary` real download, which I didn't want.

I wasn't sure if it was safe to update that check to also check for a `C:\` prefix or what else might be appropriate, so I just skipped the test on Windows.

Reply:
You can update that. 

The intent is to detect whether something is a notary channel (`nightly`, `alpha`, `beta`, `stable`) or a specific path on the file system. When I wrote it, paths always started with `./` or `/`. (even on windows, you can start them that way, powershell and go handle it)

I think it would be totally reasonable to add `c:`, `d:`, and `\` to the prefix list. It might be reasonable to look for `/` or `\` in the string, though harder to understand the points of confusion.

Reply:
All set 🎉